en/features/verification.md

```diff
@@ -0,0 +1,195 @@
+  <td><a href="/{{page.lang}}/features/details/bait-and-switch/">Bait and switch</a></td>
+  <td class="bgred"></td>
+  <td class="bgyellow"></td>
+  <td class="bggreen"></td>
+</tr>
+
+<tr>
+<td><a href="/{{page.lang}}/features/details/fabricated-transactions/">Fabricated transactions</a></td>
+  <td class="bgred"></td>
+  <td class="bgred"></td>
+  <td class="bggreen"></td>
+</tr>
+
+<tr>
+<td><a href="/{{page.lang}}/features/details/transaction-withholding/">Transaction withholding</a></td>
+  <td class="bgred"></td>
+  <td class="bgred"></td>
+  <td class="bggreen"></td>
+</tr>
+
+<tr>
+<td><a href="/{{page.lang}}/features/details/chain-hijacking/">Chain hijacking</a></td>
+  <td class="bgred"></td>
+  <td class="bgred"></td>
+  <td class="bggreen"></td>
+</tr>
+
+<tr>
+<td><a href="/{{page.lang}}/features/details/chain-rewrites/">Chain rewrites</a></td>
+  <td class="bgred"></td>
+  <td class="bgred"></td>
+  <td class="bgred"></td>
+</tr>
+
+</table>
+
+Note that although all programs---including Bitcoin Core---are
+vulnerable to chain rewrites, Bitcoin provides a defense mechanism: the
+more confirmations your transactions have, the safer you are. *There is
+no known decentralized defense better than that.*
+
+## Help Protect Decentralization
+
+{% comment %}<!-- 1231006505 is the time in block 0; 31558149 is the average number of seconds in a year -->{% endcomment %}
+{% capture bitcoin_age %}{{ site.time | date: "%s" | minus: "1231006505" | divided_by: "31558149" }}{% endcapture %}
+
+The bitcoin currency only works when people accept bitcoins in exchange
+for other valuable things. That means it's the people accepting
+bitcoins who give it value and who get to decide how Bitcoin should work.
+
+When you accept bitcoins, you have the power to enforce Bitcoin's rules,
+such as preventing confiscation of any person's bitcoins without access
+to that person's private keys.
+
+Unfortunately, many users outsource their enforcement power. This
+leaves Bitcoin's decentralization in a weakened state where a handful of
+miners can collude with a handful of service providers to change
+Bitcoin's rules for all those non-verifying users who outsourced their power.
+
+<table class="received_transactions center">
+  <tr>
+    <td class="center" markdown="span">*Users of wallet services*<br
+      >**Trust service providers**{:.fgred}</td>
+
+    <td class="center" markdown="span">*Users of P2P light wallets*<br
+    >**Trust miners**{:.fgred}</td>
+  </tr>
+
+  <tr>
+    <td class="center" markdown="span">*Users of client light wallets*<br
+    > **Trust "free" servers**{:.fgred}</td>
+
+    <td class="center" markdown="span">*Users of Bitcoin Core*<br
+    >**Enforce the rules**{:.fggreen}</td>
+  </tr>
+</table>
+
+Unlike other wallets, **Bitcoin Core *does* enforce the rules**---so
+if the miners and service providers change the rules for their non-verifying
+users, those users will be unable to pay full verification Bitcoin Core
+users like you.
```

Comment:
```suggestion
Unlike other wallets, **Bitcoin Core *does* enforce the rules**---so
if the miners and service providers change the rules for their non-verifying
users, those users will be unable to pay fully-verifying Bitcoin Core
users like you, and be kicked off the block chain.
```

Reply:
I don't know what "kicked off the block chain" means.  I think this text is fine as-is.

Reply:
They'd be forked off the chain is what I meant.

Reply:
I don't think we need to say that.  It doesn't matter what underlying data their software uses, it matters what they can or can't do, which is pay users who are still validating the rules.

Reply:
:+1: